### ISSUE TITLE
Update snapcraft core18 base to a more recent base

### ISSUE THREAD

**Issue by filfreire** [electron-userland/electron-builder#7319]
* **Electron-Builder Version**: 23.0.7

* **Node Version**: 16.17.0

* **Electron Version**: 22.0.0
* **Electron Type (current, beta, nightly)**: current

* **Target**: snap

We found [an issue](https://github.com/Kong/insomnia/issues/5531) in the [Insomnia project](https://github.com/Kong/insomnia) where the `core18` base used in the current [snapcraft.yaml](https://github.com/electron-userland/electron-builder/blob/master/packages/app-builder-lib/templates/snap/snapcraft.yaml) template is missing newer GLIBCXX libraries

```
Error: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by /run/user/1000/snap.insomnia/.org.chromium.Chromium.DdJT13)
  ```

There are options that would unblock electron-builder users bumping into similar issues:
- Allow the option to change the `base`, as requested in https://github.com/electron-userland/electron-builder/issues/5257
- Update from `core18` to a [modern base](https://snapcraft.io/docs/base-snaps#heading--supported), see PR https://github.com/electron-userland/electron-builder/pull/6976

**Comment by mmaietta** [comment]
@filfreire is this ticket still valid or was it resolved in #7320 ?

**Comment by filfreire** [comment]
@mmaietta closing it, thanks!
